.github/workflows/build_and_test.yml

```diff
@@ -858,6 +858,26 @@ jobs:
           cache-to: type=registry,ref=neondatabase/compute-node-${{ matrix.version }}:cache-${{ matrix.arch }},mode=max
           tags: |
             neondatabase/compute-node-${{ matrix.version }}:${{ needs.tag.outputs.build-tag }}-${{ matrix.arch }}
+      
+      - name: Build neon extensions test image
+        if: matrix.version == 'v16'
```

Comment:
The original task was about v16 only.
Moreover, it's redundant: we only run SQL commands over the network, why should we have all the supported Postgres versions for this?

Reply:
> The original task was about v16 only.
Wasn't it about sanitizers?
For extensions, I believe we wanted to check first how long it will take and then to decide.
I'm okay with running it on Postgres 16 only for now and later expanding it to other versions.

>  why should we have all the supported Postgres versions for this?
Several reasons:
- For some of the extensions, we use a different version for different Postgres versions
- Different Postgres versions are different, and our patches for them are different

Reply:
I agree with this, but let's implement this in another step.